readme.txt

```diff
@@ -21,6 +21,11 @@ This is why this plugin reintroduces the WordPress legacy Menus screen to be abl
 
 This also allows to use external plugins legacy widgets which might not have already been converted to blocks.
 
+== Frequently Asked Questions ==
+
+= How does it work ? =
+* Here is a video showing [how it works](https://wordpress.org/support/topic/how-it-works-56/)
```

Comment:
Could we be more precise?
Something like "How to insert a classic menu?" or "How to add a classic widget in the Site Editor?".